api/v1/test_helpers.go

```diff
@@ -11,10 +11,12 @@ import (
 )
 
 const (
-	YtsaurusName     = "test-ytsaurus"
-	CoreImageFirst   = "ytsaurus/ytsaurus-nightly:dev-23.1-5f8638fc66f6e59c7a06708ed508804986a6579f"
+	YtsaurusName = "test-ytsaurus"
+	// Images should be in sync with TEST_IMAGES variable in Makefile
+	// todo: come up with a more elegant solution
+	CoreImageFirst   = "ytsaurus/ytsaurus-nightly:dev-23.1-28ccaedbf353b870bedafb6e881ecf386a0a3779"
 	CoreImageSecond  = "ytsaurus/ytsaurus-nightly:dev-23.1-9779e0140ff73f5a786bd5362313ef9a74fcd0de"
```

Comment:
I suppose CoreImageSecond commit should be later than CoreImageFirst (we don't guarantee update with version downgrade), is it satisfied now?

(though I'm not 100% percent sure that it is requirement inside same major version, @achulkov2 can you confirm?)

Reply:
I've changed the order of the images